src/style.css

```diff
@@ -1 +1,64 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Zen+Antique+Soft&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  font-weight: 500;
+  font-size: 12px;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+}
+
+.nav_list {
+  margin-block-start: 0%;
+}
+
+.nav_item {
+  display: inline-block;
+  position: relative;
+  margin: 0 10px;
+}
+
+.item_last {
+  padding-right: 40px;
```

Comment:
remove it, you need to add paddings for both sides to the header (left and right)